js/qz-tray.js

```diff
@@ -1159,7 +1159,9 @@ var qz = (function() {
              * Any responses from the device will be sent to serial callback functions.
              *
              * @param {string} port An open port to send data over.
-             * @param {string} data The data to send to the serial device.
+             * @param {Object} data The object containing data and metadata to send to the serial device.
```

Comment:
We still need to allow for existing "string" data types too, so data type documented here needs to be `{string | Object}`